### PR TITLE
Add example use case 01

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject 

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/2262

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: examples/employees-db/service/service.bal [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        Employee employee = check dbClient->queryRow(`SELECT * FROM Employees WHERE employee_id = ${id}`);
+        return employee;
+    }
+
+    resource function post .(@http:Payload Employee emp) returns error? {
+        _ = check dbClient->execute(`
+            INSERT INTO Employees (employee_id, first_name, last_name, email, phone, hire_date, manager_id, job_title)
+            VALUES (${emp.employee_id}, ${emp.first_name}, ${emp.last_name}, ${emp.email}, ${emp.phone},
+                    ${emp.hire_date}, ${emp.manager_id}, ${emp.job_title})
+        `);
```
Comment (daneshk) [inline examples/employees-db/service/service.bal:62]: Shall we return the last inserted id here?

File: examples/employees-db/service/service.bal [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    resource function put .(@http:Payload Employee emp) returns error? {
+        _ = check dbClient->execute(`
+            UPDATE Employees
+            SET first_name = ${emp.first_name}, last_name = ${emp.last_name}, email = ${emp.email},
+                phone = ${emp.phone}, hire_date = ${emp.hire_date}, manager_id = ${emp.manager_id},
+                job_title = ${emp.job_title}
+            WHERE employee_id = ${emp.employee_id}
```
Comment (daneshk) [inline examples/employees-db/service/service.bal:71]: shall we return last updated id

File: examples/employees-db/service/service.bal [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        `);
+    }
+
+    resource function delete [int id]() returns error? {
+        _ = check dbClient->execute(`DELETE FROM Employees WHERE employee_id = ${id}`);
```
Comment (daneshk) [inline examples/employees-db/service/service.bal:76]: shall we return affected rows here

Comment (kaneeldias) [reply examples/employees-db/service/service.bal]: Fixed in cdf1da2

File: examples/employees-db/service/service.bal (hunk 2 above)
Comment (kaneeldias) [reply examples/employees-db/service/service.bal:78]: Fixed in cdf1da2

File: examples/employees-db/service/service.bal (hunk 1 above)
Comment (kaneeldias) [reply examples/employees-db/service/service.bal:63]: Fixed cdf1da2

**Review by daneshk — APPROVED** [review]

**Pull request merged**